File: src/TaskList/Models/List.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TaskList.Models
{
    public class List
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime DateCreated { get; set; }
        public bool IsDeleted { get; set; }
        public virtual List<Task> Tasks { get; set; }
    }
}

File: src/TaskList/Scripts/list.ts
export class List {
    Id: number;
    Name: string;
    IsDeleted: boolean;
    DateCreated: any;
    Tasks: any;

    constructor() {
        this.Id = null;
        this.Name = '';
        this.IsDeleted = false;
        this.DateCreated = ''
        this.Tasks = [];
       
    }
}